800.renames-and-merges/python.yaml

```diff
@@ -115,6 +115,7 @@
 - { setname: "python:khal",            name: khal }
 - { setname: "python:kiwisolver",      name: kiwisolver }
 - { setname: "python:lektor",          name: lektor }
+- { setname: "python:lesscpy",         name: lesscpy }
```

Comment:
It looks like lesscpy would be a more proper target name, as this is primarily an application and not a python module.